src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -554,7 +576,66 @@ private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColumnInfo info, string
 
                 return false;
             }
+        }
+
+        /// <summary>
+        /// An interface implemented by items of <see cref="ColumnFunctions"/> corresponding to the
+        /// <see cref="NormalizeTransform.AffineColumnFunction"/> items.
+        /// </summary>
+        internal interface IAffineData<TData>
+        {
+            /// <summary>
+            /// The scales. In the scalar case, this is a single value. In the vector case this is of length equal
+            /// to the number of slots. Function is <c>(input - offset) * scale</c>.
+            /// </summary>
+            TData Scale { get; }
+
+            /// <summary>
+            /// The offsets. In the scalar case, this is a single value. In the vector case this is of length equal
+            /// to the number of slots, or of length zero if all the offsets are zero.
+            /// </summary>
+            TData Offset { get; }
+        }
 
+        /// <summary>
+        /// An interface implemented by items of <see cref="ColumnFunctions"/> corresponding to the
+        /// <see cref="NormalizeTransform.CdfColumnFunction"/> items. The function is the value of the
+        /// cumulative density function of the normal distribution parameterized with mean <see cref="Mean"/>
+        /// and standard deviation <see cref="Stddev"/>.
+        /// </summary>
+        internal interface ICdfData<TData>
+        {
+            /// <summary>
+            /// The mean(s). In the scalar case, this is a single value. In the vector case this is of length equal
+            /// to the number of slots.
+            /// </summary>
+            TData Mean { get; }
+
+            /// <summary>
+            /// The standard deviation(s). In the scalar case, this is a single value. In the vector case this is of
+            /// length equal to the number of slots.
+            /// </summary>
+            TData Stddev { get; }
+
+            /// <summary>
+            /// Whether the input should first be considered to be logged, that is, whether this was configured
+            /// over a log-normal as opposed to regular normal distribution.
+            /// </summary>
+            bool UseLog { get; }
```

Comment:
>UseLog [](start = 17, length = 6)

If I correctly understand you need this interface to work with OnFit delegate, but in same time I don't see UseLog been passed to OnFit.
Do you need this UseLog?

Reply:
I'd think not, but let me know if you disagree. That boolean is a constant that is passed in *precisely* the same location that we pass this selfsame delegate in -- so I'm not sure I see the point in cluttering the signature of the delgate (I mean, heck, how about I pass along max-num-instances while I'm at it). I don't feel like I'd need it, as a practical matter in the delegate.